docs/guides/tutorials/trpc.rst

```diff
@@ -0,0 +1,468 @@
+           <p key={user.email}>{user.name}</p>
+         ))}
+       </div>
+     );
+   };
+
+   export default UsersComponent;
+
+Alternative Path: Use tRPC with Express
+---------------------------------------
+
+If you're not using **Next.js**, here's how you can integrate **tRPC** with
+**Express**.
+
+Set up Express server with tRPC
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here’s how you can create an Express server and integrate **tRPC**:
+
+.. code-block:: typescript
+
+   import express from 'express';
+   import { appRouter } from './routers/_app';
+   import * as trpcExpress from '@trpc/server/adapters/express';
+
+   const app = express();
+
+   app.use(
+     '/trpc',
+     trpcExpress.createExpressMiddleware({
+       router: appRouter,
+     })
+   );
+
+   app.listen(4000, () => {
+     console.log('Server is running on port 4000');
+   });
+
+Client-side usage
+-----------------
+
+In non-Next.js apps, use the tRPC client to interact with the server:
+
+.. code-block:: typescript
+
+   import { createTRPCClient, httpBatchLink } from '@trpc/client';
+   import { AppRouter } from './routers/_app';
+
+   const trpc = createTRPCClient<AppRouter>({
+     links: [
+       httpBatchLink({
+         url: 'http://localhost:4000/trpc',
+       }),
+     ],
+   });
+
+   async function fetchUsers() {
+     const users = await trpc.getUsers.query();
+     console.log(users);
+   }
+
+Step 5: Set up authentication with EdgeDB Auth
+==============================================
+
+In this section, we will cover how to integrate **EdgeDB Auth** with **tRPC** 
+and context in both **Next.js** and **Express** environments. This will ensure 
+that user authentication is handled securely and that both server-side and 
+client-side tRPC calls can access the user’s session.
+
+EdgeDB Auth with tRPC and tRPC context in Next.js
+-------------------------------------------------
+
+In **Next.js**, integrating **EdgeDB Auth** with **tRPC** involves creating a 
+context that provides the user session and EdgeDB client to the tRPC API.
+
+1. **Initialize EdgeDB Client and Auth**
+
+   First, initialize the **EdgeDB** client and **EdgeDB Auth**:
+
+   .. code-block:: typescript
+
+      import { createClient } from "edgedb";
+      import createAuth from "@edgedb/auth-nextjs/app";
+
+      // Initialize EdgeDB client
+      export const edgedbClient = createClient();
+
+      // Initialize EdgeDB Auth
+      export const auth = createAuth(edgedbClient, {
+        baseUrl: process.env.VERCEL_ENV === "production"
+          ? "https://production.yourapp.com"
+          : "http://localhost:3000",
+      });
+
+2. **Create tRPC Context**
+
+   The **tRPC** context provides the session and the EdgeDB client to the tRPC 
+   procedures:
+
+   .. code-block:: typescript
+
+      import { initTRPC } from '@trpc/server';
+      import { headers } from "next/headers";
+      import { auth } from "./path-to-auth";
+      import { edgedbClient } from "./path-to-edgedb";
```

Comment:
These are coming from the same module, I think, so if we make the module location and name explicit above, we can make this more explicit.